s3_transport/include/irods/private/s3_transport/s3_transport.hpp

```diff
@@ -552,12 +553,12 @@ namespace irods::experimental::io::s3_transport
 
                 return shm_obj.atomic_exec([this, _buffer, _buffer_size](auto& data) {
 
-                    auto position_before_write = this->cache_fstream_.tellp();
+                    std::streamoff position_before_write = this->cache_fstream_.tellp();
                     this->cache_fstream_.write(_buffer, _buffer_size);
                     this->cache_fstream_.flush();
 
                     // calculate the new size of the file
-                    auto current_position = this->cache_fstream_.tellp();
+                    std::streamoff current_position = this->cache_fstream_.tellp();
 
                     const auto msg = fmt::format("send() position={} size={} position_after_write=", position_before_write, _buffer_size, current_position);
```

Comment:
Are we missing a set of curly braces here? `msg` has 2 sets of curly braces, but is taking 3 parameters. I think this should be...
```suggestion
                    const auto msg = fmt::format("send() position={} size={} position_after_write={}", position_before_write, _buffer_size, current_position);
```

Reply:
nice catch

Reply:
Not sure if I'm going to handle it in this PR, but I've opened an issue: #2266

Leaving this unresolved until I decide

Reply:
Resolving this